cli/cmd/sync_test.go

```diff
@@ -66,6 +66,16 @@ func TestSync(t *testing.T) {
 			logContent := string(b)
 			require.NoError(t, logFileError, "failed to read cloudquery.log")
 			require.NotEmpty(t, logContent, "cloudquery.log empty; expected some logs")
+
+			logFileName = path.Join(cqDir, "cloudquery1.log")
```

Comment:
Maybe we should use the same log file in this command, otherwise it won't get closed by the `defer CloseLogFile()` call above